src/data/sign.lean

```diff
@@ -228,3 +225,6 @@ begin
 end
 
 end add_group
+
+theorem int.not_acc (a : ℤ) : ¬ acc (<) a :=
```

Comment:
This definitely doesn't belong in this PR, or even this file

Reply:
Ah, my bad. I wrote it up as an example on Zulip and accidentally added it to the PR.